lib/twitter-ads/creative/promoted_tweet.rb

```diff
@@ -18,10 +18,10 @@ class PromotedTweet
       property :created_at, type: :time, read_only: true
       property :updated_at, type: :time, read_only: true
       property :deleted, type: :bool, read_only: true
+      property :paused, type: :bool, read_only: true
```

Comment:
Correct that this is read-only, but the attribute name was changed to `entity_status`:

```
$ date && twurl -H ads-api.twitter.com "/2/accounts/18ce54d4x5t/promoted_tweets?count=1"
Thu Dec  7 23:35:43 PST 2017
{
  "request": {
    "params": {
      "count": 1,
      "account_id": "18ce54d4x5t"
    }
  },
  "next_cursor": "8sxrn1xxc",
  "data": [
    {
      "line_item_id": "5tddn",
      "id": "x73d7",
      "entity_status": "ACTIVE",
      "created_at": "2016-07-29T22:25:45Z",
      "updated_at": "2016-12-05T23:33:37Z",
      "approval_status": "ACCEPTED",
      "tweet_id": "759137621666766848",
      "deleted": false
    }
  ]
}
```